vita3k/main.cpp

```diff
@@ -136,8 +136,24 @@ int main(int argc, char *argv[]) {
         auto discord_rich_presence_old = host.cfg.discord_rich_presence;
 #endif
 
+        std::chrono::system_clock::time_point present = std::chrono::system_clock::now();
+        std::chrono::system_clock::time_point later = std::chrono::system_clock::now();
+        const double frame_time = 16.67;
```

Comment:
const double frame_time = 1000.0 / 60.0;

You make 60.0 a constant if you want.

Reply:
Fixed